app/src/main/java/mozilla/org/webmaker/MainActivity.java

```diff
@@ -24,6 +25,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        Log.v("[Webmaker]", "MainActivity created.");
```

Comment:
Yeah... I like having lifecycle events in the ADB logs. Happy to remove these types of logs if you prefer.